public/js/app/pages/CreateNewMemo.jsx

```diff
@@ -129,6 +129,24 @@ function CreateNewMemo(props) {
     return { memoCommonLangAbbr, languageOfInstructions }
   }
 
+  const _getSortedCourseRoundsIds = (term, applicationCodes, courseRoundTerms) => {
```

Comment:
When application codes are added, then we dont need anymore `ladokRoundIds` so you can delete all code connected with it

Reply:
Removed.